sbi/utils/plot.py

```diff
@@ -1,809 +0,0 @@
-# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
```

Comment:
is this a rebase artefact or was this still existing in `main`?

Reply:
can't find it in main, so probably some rebase artefact